provision-virtualization-tools.sh

```diff
@@ -59,17 +62,17 @@ rm $terraform_libvirt_provider_filename
 
 # install Packer.
 apt-get install -y unzip
-packer_version=1.5.5
+packer_version=1.6.6
 wget -q -O/tmp/packer_${packer_version}_linux_amd64.zip https://releases.hashicorp.com/packer/${packer_version}/packer_${packer_version}_linux_amd64.zip
-unzip /tmp/packer_${packer_version}_linux_amd64.zip -d /usr/local/bin
+yes | unzip /tmp/packer_${packer_version}_linux_amd64.zip -d /usr/local/bin
```

Comment:
can `yes |` be replaced with the the unzip `-o` option instead?

Reply:
Also you rigth, I just wanted to change the less as possible

Reply:
The second time you run the script, It asks for confirmation, first time dont